src/shared/ui/form/form.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { cleanup, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { describe, it, vi, expect, afterEach, beforeEach } from 'vitest'
+import z from 'zod'
```

Comment:
최근에 과제 진행하면서 애널라이저(빌드된 파일, 폴더, 패키지 크기 보는 라이브러리)를 활용했는데, 

위와 같이 작성하면 번들크기가 엄청나더라고요.. 따라서 다음과 같이 변경해주시는게 좋아보입니다. 
이건 zod 공식문서에서 확인했습니다!

https://zod.dev/packages/mini

```suggestion
import * as z from 'zod';
```